mapping/PANCAN-mutation/map-mutations.ipynb

```diff
@@ -0,0 +1,515 @@
+       "      <td>A1B|ABG|GAB|HYST2477</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>2</td>\n",
+       "      <td>12</td>\n",
+       "      <td>A2M</td>\n",
+       "      <td>A2MD|CPAMD5|FWP007|S863-7</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   entrez_id chr symbol                   Synonyms\n",
+       "0          1  19   A1BG       A1B|ABG|GAB|HYST2477\n",
+       "1          2  12    A2M  A2MD|CPAMD5|FWP007|S863-7"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "base_url = 'ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Mammalia/'\n",
+    "map_name = 'Homo_sapiens.gene_info.gz'\n",
+    "map_url = base_url + map_name\n",
+    "path = os.path.join('download', map_name)\n",
+    "urlretrieve(map_url, path)\n",
+    "\n",
+    "path = os.path.join('download', map_name)\n",
+    "\n",
+    "map_df = (\n",
+    "    pandas.read_table(path)\n",
+    "    .rename(columns={'#tax_id' : 'tax_id', 'GeneID': 'entrez_id', 'chromosome':'chr', 'Symbol':'symbol'})\n",
+    ")\n",
+    "\n",
+    "#filter to include only tax_id == 9606 (human) and columns of interest\n",
+    "map_df = map_df.ix[map_df['tax_id'] == 9606, ['entrez_id','chr', 'symbol', 'Synonyms']]\n",
+    "\n",
+    "map_df.head(2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>entrez_id</th>\n",
+       "      <th>chr</th>\n",
+       "      <th>symbol</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>1</td>\n",
+       "      <td>19</td>\n",
+       "      <td>A1BG</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>2</td>\n",
+       "      <td>12</td>\n",
+       "      <td>A2M</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   entrez_id chr symbol\n",
+       "0          1  19   A1BG\n",
+       "1          2  12    A2M"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "map_primary = map_df[['entrez_id','chr', 'symbol']]\n",
+    "map_primary.head(2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "alternates = map_df['Synonyms'].str.split('|').apply(pandas.Series, 1).stack()\n",
```

Comment:
Maybe add a comment above this line about what this function is doing? We had some difficulty figuring it out